packages/apple-pay-integration/src/mocks/apple-pay-button.mock.ts

```diff
@@ -1,15 +1,199 @@
+        couponAmount: 0,
+        listPrice: 200,
+        salePrice: 200,
+        comparisonPrice: 200,
+        downloadPageUrl: 'url.php',
+        downloadFileUrls: [],
+        downloadSize: '',
+        extendedListPrice: 200,
+        extendedSalePrice: 200,
+        extendedComparisonPrice: 200,
+        addedByPromotion: false,
+        options: [
+            {
+                name: 'm',
+                nameId: 1,
+                value: 'l',
+                valueId: 3,
+            },
+        ],
+        categories: [[{ name: 'Cat 1' }], [{ name: 'Cat 2' }], [{ name: 'Cat 3' }]],
+        categoryNames: ['Ebooks', 'Audio Books'],
+    };
+}
+
+export function getGiftCertificateItem(): GiftCertificateItem {
+    return {
+        id: 'bd391ead-8c58-4105-b00e-d75d233b429a',
+        name: '$100 Gift Certificate',
+        message: 'message',
+        amount: 100,
+        taxable: false,
+        theme: 'General',
+        sender: {
+            name: 'pablo',
+            email: 'pa@blo.com',
+        },
+        recipient: {
+            name: 'luis',
+            email: 'lu@is.com',
+        },
+    };
+}
+
+export function getPhysicalItem(): PhysicalItem {
+    return {
+        id: '666',
+        variantId: 71,
+        productId: 103,
+        sku: 'CLC',
+        name: 'Canvas Laundry Cart',
+        url: '/canvas-laundry-cart/',
+        quantity: 1,
+        brand: 'OFS',
+        isTaxable: true,
+        imageUrl: '/images/canvas-laundry-cart.jpg',
+        discounts: [],
+        discountAmount: 10,
+        couponAmount: 0,
+        listPrice: 200,
+        salePrice: 190,
+        comparisonPrice: 200,
+        extendedListPrice: 200,
+        extendedSalePrice: 190,
+        extendedComparisonPrice: 200,
+        isShippingRequired: true,
+        addedByPromotion: false,
+        options: [
+            {
+                name: 'n',
+                nameId: 1,
+                value: 'v',
+                valueId: 3,
+            },
+        ],
+        categories: [[{ name: 'Cat 1' }], [{ name: 'Furniture' }, { name: 'Bed' }]],
+        categoryNames: ['Cat 1'],
+    };
+}
+export const getCart = () => {
+    return {
+        id: 'b20deef40f9699e48671bbc3fef6ca44dc80e3c7',
+        customerId: 4,
+        currency: getCurrency(),
+        email: 'foo@bar.com',
+        isTaxIncluded: false,
+        baseAmount: 200,
+        discountAmount: 10,
+        cartAmount: 190,
+        coupons: [getCoupon(), getShippingCoupon()],
+        discounts: [getDiscount()],
+        lineItems: {
+            physicalItems: [getPhysicalItem()],
+            digitalItems: [getDigitalItem()],
+            giftCertificates: [getGiftCertificateItem()],
+            customItems: [],
+        },
+        createdTime: '2018-03-06T04:41:49+00:00',
+        updatedTime: '2018-03-07T03:44:51+00:00',
+    };
+};
+
+export const buyNowCartMock = {
+    ...getCart(),
+    id: 999,
+    source: CartSource.BuyNow,
+};
+
```

Comment:
You can use all mocked objects from `bigcommerce/payment-integrations-test-utils` like `getBuyNowCart`, `getCart` and others

Reply:
fixed